transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/UpdateItemTransformerTest.java

```diff
@@ -103,9 +107,9 @@ void testTransformJaxbToDomainResubmitMessage() {
         final IIndividualRequest domainObject = updateItemTransformer.transformJaxbToDomain(updateRequestType);
 
         assertNotNull(domainObject);
-        assertEquals(sdtRequestId, domainObject.getSdtRequestReference(), "Found correct sdt request id");
-        assertEquals(UpdateStatusCodeType.RESUBMIT_MESSAGE.value(), domainObject.getRequestStatus(),
-                "Found correct request status");
+        assertEquals(domainObject.getSdtRequestReference(), sdtRequestId, FOUND_CORRECT_SDT_REQUEST_ID);

```

Comment:
First two parameters are around the wrong way.  The first should be the expected value and the second the actual value.

Reply:
assertEquals(SDT_REQUEST_ID, domainObject.getSdtRequestReference(), FOUND_CORRECT_SDT_REQUEST_ID);

Reply:
Confirmed parameters have been corrected.